dv/uvm/yaml/rtl_simulation.yaml

```diff
@@ -19,6 +19,7 @@
          -l <out>/compile.log
          -sverilog -ntb_opts uvm-1.2
          +define+UVM_REGEX_NO_DPI -timescale=1ns/10ps -licqueue
+         +incdir+../../shared/rtl
```

Comment:
Can we put that option onto the ibex_core.f file to make all simulators use it?

Reply:
Sure, I can do that